examples/special-workload.yaml

```diff
@@ -35,7 +35,7 @@ spec:
         ports:
         - name: http
           containerPort: 80
-          hostPort: 8082
+          hostPort: 8097 
```

Comment:
This again is a port number inside the kind cluster rather than on the host of the kind cluster, so stays 8082.